section4/exercises/burrito.rb

```diff
@@ -11,9 +11,61 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
-end
+
+  def add_topping
```

Comment:
For each of these methods, we want you to use arguments (so passing in a topping or protein) and then updating the instance variable's value. You should not need to use loops in #add_topping.